mobileapp/www/index.html

```diff
@@ -172,56 +172,47 @@
 
 
       <div id="onboarding-no-account-step-2" class="view intro-graphics">
-	<div class="logo">
-	  <img class="company-logo" src="img/spideroak-white.png" />
-	  <img src="svg/kloak-logo.svg" />
-	  <!-- <h1>Kloak</h1> -->
-	</div>
+	<div class="onboarding-header-container">
+      <img src="svg/kloak-full-logo.svg" />
+  </div>
 
 	<div id="onboarding-create-passphrase">
 	  <h3>Passphrase:</h3>
-
+    <a href="#"
+       id="regenerate-passphrase"
+       class="onboarding-link">Regenerate</a>
 	  <div id="passphrase-info">
 	    <textarea placeholder="Enter Passphrase"
 		      autocapitalize="off"
-		      rows="3"
+		      rows="2"
 		      maxlength="256"
 		      class="onboarding-input"
 		      id="onboarding-passphrase-input"></textarea>
-	    <div class="onboarding-continue-wrapper">
-	      <button class="btn onboarding-btn"
-		      id="onboarding-create-account">Continue</button>
-	      <p>
-		<a href="#"
-		   id="regenerate-passphrase"
-		   class="onboarding-link">Regenerate Passphrase</a>
-	      </p>
-	      <p>
-		<a href="#"
-		   id="clear-passphrase"
-		 class="onboarding-link">I'll choose my own Passphrase</a>
-	      </p>
-	    </div>
-	    <a href="#" class="onboarding-link" id="onboarding-cancel">Cancel</a>
-	    <p class="onboarding-light-copy">
-	      We have created a strong passphrase for you, please save it into your password wallet or write it down and put it in a safe place! You cannot ask SpiderOak to recover a forgotten passphrase. We <strong>do not</strong> store it outside the device.
-	    </p>
+    <a href="#"
+       id="clear-passphrase"
+     class="onboarding-link">I'll choose my own Passphrase</a>
+      <p class="onboarding-light-copy">
+        You will only be asked for your passphrase when you sign out of Kloak, or sign in on a new device.
```

Comment:
When you sign out of Kloak your passphrase is saved in "Secure Storage" (KeyChain, etc) on Android 5+ and iOS 8+, so this is not so accurate for what I imagine to be a large number of users. Perhaps we should come up with better explainer here?